src/plugins/share/public/types.ts

```diff
@@ -86,6 +90,15 @@ export interface ShareMenuProvider {
   readonly id: string;
 
   getShareMenuItems: (context: ShareContext) => ShareMenuItem[];
+
+  jobProviderOptions:
```

Comment:
Looks like this should not be added. I don't see where the Sharing services use this